aspen/resources/__init__.py

```diff
@@ -97,6 +97,52 @@ def get_declaration(line):
     return fulltext.decode(encoding or b'ascii')
 
 
+
+class SimplateWrapper(Simplate):
```

Comment:
Okay. This is made clearer in https://github.com/gratipay/aspen-python/commit/9d57bd70e9bb992fff6c9c3c616e0e823892b395 and https://github.com/gratipay/aspen-python/commit/b52ca20a1eb779327998a1b5dd4d39c10c9cbe6d, where I've refactored `SimplateWrapper` and `StaticResource` into `aspen.http.resource.Dynamic` and `.Static`.